bwc-test/build.gradle

```diff
@@ -76,13 +76,13 @@ dependencies {
 String bwcVersion = "1.3.0.0";
 String baseName = "securityBwcCluster"
 String bwcFilePath = "src/test/resources/"
-String projectVersion = "2.0.0.0"
```

Comment:
Can we pull this from the other gradle configuration? so we don't have to keep updating both files?

Reply:
This is the only occurrence. We don't have to pull it to other places.

Reply:
> This is the only occurrence.

How is this the case? This pull request has effectively the same change at build.gradle@60, bwc-test/build.gradle@50 and bwc-test/build.gradle@79

Reply:
I see what you're referring to now. I thought that you were referring to this line#79 specifically. This is security plugin version. The others are OpenSearch version.
The 2 files are in 2 project folders. `bwc-test` is a sub-project of security plugin project. I don't see how the versions in 2 different project can be extracted to a central place though.

Reply:
Let's table this for after we've correctly rewritten the build.gradle https://github.com/opensearch-project/security/issues/1839